vue/src/views/Crops3dView.vue

```diff
@@ -1,23 +1,33 @@
 <template>
-  <div class="ml-28">
-    <header-bar
-      title="3D"
-      :widgetOptions="widgetOptions3D"
-      :storeOptions="storeOptions"
-    ></header-bar>
-    <div class="row p-6">
-      <div v-if="storeOptions.indexOf('3d-table') > -1" class="col-4">
-        <crops-table title="Plants" :visibleColumns="columns"></crops-table>
-      </div>
-      <div v-if="storeOptions.indexOf('3d-map') > -1" class="col-8 pl-2">
-        <crops-map></crops-map>
-      </div>
+  <header-bar
+    title="3D"
+    :widgetOptions="widgetOptions3D"
+    :storeOptions="storeOptions"
+  ></header-bar>
+  <div class="row p-6">
+    <div v-if="storeOptions.indexOf('3d-table') > -1" class="col-4">
+      <crops-table
+        ref="cropsTableRef"
+        title="Plants"
+        :visibleColumns="columns"
+        :crops="crops"
+        @row-enter="tableCropsEnter"
+        @row-leave="tableCropsLeave"
+      ></crops-table>
+    </div>
+    <div v-if="storeOptions.indexOf('3d-map') > -1" class="col-8 pl-2">
+      <crops-map
+        ref="cropsMapRef"
+        :crops="crops"
+        @polygon-enter="mapCropsEnter"
+        @polygon-leave="mapCropsLeave"
+      ></crops-map>
     </div>
   </div>
 </template>
 
 <script setup lang="ts">
-import { onMounted, type Ref } from "vue";
+import { onMounted, type Ref, ref } from "vue";
```

Comment:
removed unused import
```suggestion
import { type Ref, ref } from "vue";
```